### PR TITLE
New futures controller

### PR DESCRIPTION
**Pull request by DidierRLopes**
* Features working ✅
* Docs ✅
* Unit Tests ✅
* Integration Tests ✅
* SDK ✅

### CODE REVIEW

**Comment by jmaslek** [comment]
On `search`, can we search by description as well?  This way I could do like `search -d e-mini`

File: openbb_terminal/futures/futures_controller.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            prog="historical",
+            description="""Display futures historical. [Source: YahooFinance]""",
+        )
+        parser.add_argument(
+            "-t",
+            "--ticker",
+            dest="ticker",
+            type=str,
+            default="",
+            help="Future ticker to display timeseries separated by comma when multiple, e.g.: BLK,ES",
```
Comment (jmaslek) [inline openbb_terminal/futures/futures_controller.py:155]: Note ES isnt in the futures.csv

**Comment by jmaslek** [comment]
> `curve 10Y`. Should check that there is data before plotting.
> 
> Screen Shot 2022-10-18 at 10 06 47 AM

i did this 4 u

**Comment by jmaslek** [comment]
`curve KC`

x axis formatting

Screen Shot 2022-10-18 at 10 14 39 AM

**Comment by DidierRLopes** [comment]
> On `search`, can we search by description as well? This way I could do like `search -d e-mini`

Screenshot 2022-10-18 at 18 58 19

**Comment by DidierRLopes** [comment]
> > `curve 10Y`. Should check that there is data before plotting.
> > Screen Shot 2022-10-18 at 10 06 47 AM
> 
> i did this 4 u

Screenshot 2022-10-18 at 18 59 32

Wth did you do?

**Comment by jmaslek** [comment]
> > > > `curve 10Y`. Should check that there is data before plotting.
> > > > Screen Shot 2022-10-18 at 10 06 47 AM
> > > 
> > > 
> > > i did this 4 u
> > 
> > 
> > Screenshot 2022-10-18 at 18 59 32
> > Wth did you do?
> 
> That case was handled. How did u manage to get that? Screenshot 2022-10-18 at 19 00 38

I added that case earlier.  I pushed to your branch. look at commits :D

**Comment by jmaslek** [comment]
now if i do curve 10Y it tells me no data and pops up an empty plot

**Comment by deeleeramone** [comment]
A little funny business with the autocomplete. 

Screen Shot 2022-10-18 at 6 21 08 PM

**Comment by deeleeramone** [comment]
For some reason, `ES=F,E-mini S&P 500 Index Futures,CME,index` is missing from the list.

**Comment by deeleeramone** [comment]
There should be a way to display the continuous chart, which is the `=f` version of the symbol. 

Screen Shot 2022-10-18 at 6 39 25 PM

A useful number to know is the spread between the front month contract (which should always be the `=f` symbol) and the forward month(s). Front month > forward month = backwardation; bullish for commodities, but not for interest rates and indexes. Front month < forward month = bullish for stocks, indexes and a "normal" interest rate yield curve.

**Comment by DidierRLopes** [comment]
@jmaslek Tests added!! 🚀

**Comment by DidierRLopes** [comment]
@Chavithra @colin99d will need your help on this Monday please! 🙏🏽

**Pull request merged**